MapleServer2/Types/Party.cs

```diff
@@ -0,0 +1,70 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Servers.Game;
+using System;
+using System.Collections.Generic;
+
+namespace MapleServer2.Types
+{
+    public class Party
+    {
+        public int Uid { get; private set; }
+        public int MaxMembers { get; set; }
+        public Player Leader { get; set; }
+
+        //List of players and their session.
+        public HashSet<Player> Players { get; private set; }
+
+        public Party(int pUid, int pMaxMembers, Player pLeader, HashSet<Player> pPlayers)
```

Comment:
Can the leader be added to the players in the constructor? that way we don't need to pass the leader twice, just the leader, then the other members?

Basically makes
`Party newParty = new Party(GuidGenerator.Int(), 10, partyLeader, new HashSet<Player> { partyLeader, session.Player });`
turn into
`Party newParty = new Party(GuidGenerator.Int(), 10, partyLeader, new HashSet<Player> { session.Player });`

Reply:
Instead of passing partyLeader separately, I can just set the party leader to the first player in the list. Easier than prepending the party leader.